libs/motr/emap_fi_adapter.py

```diff
@@ -183,13 +199,97 @@ def get_object_cob_id(self, oid, dtype):
         """
         return ''
 
-    def get_metadata_shard(self, oid):
+    # pylint: disable-msg=too-many-locals
+    def get_object_gob_id(self, metadata_device, parse_size=PARSE_SIZE, fid: dict = None):
+        """
+        Fetch COB ID from the M0CP trace file.
+        :param metadata_device:
+        :param parse_size:
+        :param fid: dict of object id of data and parity block
+        :return: FID to be corrupted
+        """
+        pod_list = self.master_node_list[0].get_all_pods(POD_NAME_PREFIX)
+        LOGGER.debug("pod list is %s", pod_list)
+        d_fid = []
```

Comment:
optionally we can rename this var name to be more elaborate like `data_fid` and `parity_fid`